src/actions/registrationActions.js

```diff
@@ -162,7 +162,10 @@ export const submitForm = (state, id) => {
             const course = formatCourse(state, 'class');
 
             for (const key in course) {
```

Comment:
I know you didn't create this, but since there was a change here and I read it lol: Create a helper/explanatory function that documents this loop better
```
const removeUndefinedCourseProperties = (courseFormData) => {
    // insert for loop + if statement code here
}